charts/kueue/templates/rbac/topology_viewer_role.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: '{{ include "kueue.fullname" . }}-topology-viewer-role'
+  labels:
+  {{- include "kueue.labels" . | nindent 4 }}
+    rbac.kueue.x-k8s.io/batch-admin: "true"
+    rbac.kueue.x-k8s.io/batch-user: "true"
```

Comment:
I think we should restrict it only to admins, by analogy to the ResourceFlavor: https://github.com/kubernetes-sigs/kueue/blob/8b63e005815d8dd48d4d38de0ffe9472d29507c5/charts/kueue/templates/rbac/resourceflavor_viewer_role.yaml#L8.

Note that the Topology levels are already exposed to users via the LocalQueue.status. It is no ideal, but I would keep this approach for consistency for now.

Reply:
Ack